dandi/cli/cmd_ls.py

```diff
@@ -358,7 +358,20 @@ def fn():
                             digest=Digest.dandi_etag(digest),
                         ).json_dict()
                 else:
-                    rec = get_metadata(path)
+                    if path.endswith(tuple(ZARR_EXTENSIONS)):
+                        if use_fake_digest:
+                            digest = "0" * 32 + "-1"
```

Comment:
```suggestion
                            digest = "0" * 32 + "-0--0"
```

So that it has the form of a Zarr digest.